app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-form-tracker.js

```diff
@@ -8,6 +8,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.module = module
     this.trackingTrigger = 'data-ga4-form' // elements with this attribute get tracked
     this.includeTextInputValues = this.module.hasAttribute('data-ga4-form-include-text')
+    this.recordJson = this.module.hasAttribute('data-ga4-form-record-json')
```

Comment:
I'd like to understand more about this change - specifically how the new output differs from the old and what the existing code lacked. Also if this is useful is it not worth making it the default behaviour, was this discussed with the other PAs?

Reply:
That's a really helpful summary, thanks. Could you add that into the docs?

Reply:
I've added that to docs in the commit that adds this change 🙂